lib/controllers/include/Current.h

```diff
@@ -6,12 +6,12 @@ namespace Controller {
 class Current final : public ActuatorController {
  public:
   Current(Actuator::Actuator &actuator, Encoder::Encoder &encoder,
-          const std::optional<std::reference_wrapper<Sensor::CurrentSensor> const> &currentSensor, PID::PID &pid,
+          Sensor::CurrentSensor &currentSensor, PID::PID &pid,
           float maxDegPerSec, float maxCurrent, PinName lowerLimit, PinName upperLimit,
           bool ignoreDegPerSecChecks = false, bool ignoreCurrentChecks = true, bool ignoreLimitSwitchChecks = false);
   bool update() override;
   void stop() override;
-  void reset() override;
+  bool reset() override;
```

Comment:
reset functions of controllers need to return a bool since encoder.reset returns a bool